src/components/Button.js

```diff
@@ -0,0 +1,5 @@
+function Button(props) {
+  return <h2>{props.name}</h2>;
+}
```

Comment:
- [ ] Use es6 function syntax here.

- [ ] Also please validate propTypes in this component.